content/pages/software/grand-challenge.md

```diff
@@ -4,59 +4,72 @@ finished: false
 picture: software/grand-challenge.png
 template: project-single
 groups: rse, diag, rtc
-people: James Meakin, Sjoerd Kerkstra, Paul Konstantin Gerke, Harm van Zeeland, Bram van Ginneken, Sil van de Leemput, Mike Overkamp, Miriam Groeneveld, Anne Mickan, Chris van Run, Kiran Vaidhya Venkadesh
+people: James Meakin, Henkjan Huisman, Sjoerd Kerkstra, Paul Konstantin Gerke, Harm van Zeeland, Miriam Groeneveld, Anne Mickan, Chris van Run, Thomas Koopman 
 description: The home of challenges in biomedical imaging.
 
 bibkeys:
-github: https://github.com/comic/grand-challenge.org
+github: https://github.com/DIAGNijmegen/grand-challenge
 
+We develop and maintain [grand-challenge.org](https://grand-challenge.org), an open-source, Django-based platform for hosting biomedical imaging challenges, collecting expert annotations, and making deep learning models accessible to clinicians and researchers worldwide.
```

Comment:
I'd lose the "Django-based" here. The tech stack is already mentioned below and it's not that important.